Readme.md

```diff
@@ -252,3 +252,23 @@ then exit.
                         Optional. Print the level of compression for WIA/RVZ
                         formats, then exit.
 ```
+
+```
+Usage: extract [options]...
+
+Options:
+  -h, --help            show this help message and exit
+  -i FILE, --input=FILE
+                        Path to disc image FILE. *
+  -o FOLDER, --output=FOLDER
+                        Path to the destination FOLDER. *
+  -p PARTITION, --partition=PARTITION
+                        Which specific partition you want to extract.
+  -s SINGLE, --single=SINGLE
+                        Which specific file/directory you want to extract.
+  -l LIST, --list=LIST  List all files in volume/partition and print it out to
+                        a text file (ls.txt).
+  -m, --mute            Mute all messages except for errors.
```

Comment:
I usually see these as `-q`/`--quiet` for no output, no flag for low output, `-v`/`--verbose` for high output; `-m`/`--mute` feels rather nonstandard. Any particular reason you want for this?

Reply:
Nope! I'm just terrible at naming things. I've changed the flag to 
 ```-q```/```--quiet```